addon-test-support/index.js

```diff
@@ -4,7 +4,7 @@ import { assert } from '@ember/debug';
 export async function upload(selector, file) {
   let input = find(selector);
   assert(`Selector '${selector}' is not input element.`, input.tagName === 'INPUT');
-  assert(`File must be instance of File type`, file instanceof File);
+  assert(`File must be instance of File type`, file instanceof File || file instanceof Blob);
```

Comment:
Maybe change the assertion to mention `Blob`?

`File must be instance of File or Blog type`?

Reply:
You could also just check if `file instanceof Blob` since File is based on Blog:
https://developer.mozilla.org/en-US/docs/Web/API/File